lib/components/atoms/Button/Button.js

```diff
@@ -30,11 +33,13 @@ const Button = ({
   disabled,
   ariaLabel,
   elementLocator,
+  icon,
```

Comment:
This is not correct, we should we Closed for Modification and Open for Enhancement